assignments/Session_03/wsgi-calc/calculator.py

```diff
@@ -0,0 +1,215 @@
+import re
+import sys
+
+
+def Instruction():
+    page = """
+<h1>"Python Calculator Project"</h1>
+
+<p> <font face="Arial", font size="+2">
+Addition<br></font>
+<font face="Arial", font size="+1">
+expected format:<br>
+/addition/##/##/## where ## is a number - One to many numbers can be provided<br>
+click the example ->   <a href="/addition/4/5">/addition/4/5</a><br><br></font>
+
+<p> <font face="Arial", font size="+2">
+Subtraction<br></font>
+<font face="Arial", font size="+1">
+expected format:<br>
+/subtraction/##/##/## where ## is a number - One to many numbers can be provided<br>
+Each number is subtracted from the prior number to produce the final value<br>
+click the example ->   <a href="/subtraction/15/5">/subtraction/15/5</a><br><br></font>
+
+<p> <font face="Arial", font size="+2">
+Multiplication<br></font>
+<font face="Arial", font size="+1">
+expected format:<br>
+/multiplication/##/##/## where ## is a number - One to many numbers can be provided<br>
+click the example ->   <a href="/multiplication/3/8/2">/multiplication/3/8/2</a><br><br></font>
+
+<p> <font face="Arial", font size="+2">
+Division<br></font>
+<font face="Arial", font size="+1">
+expected format:<br>
+/division/##/##/## where ## is a number - One to many numbers can be provided<br>
+The numbers are divided in the order provided<br>
+Any "divide by zero" events will prompt an exception<br>
+click the example ->   <a href="/division/20/5/3">/division/20/5/3</a><br><br></font>
+"""
+    return page.format()
+
+
+def addition(*args):
+    page = """
+<h1>Calculator - Addition</h1>
+<p> <font face="Arial", font size="+1">
+Operands provided: {0}<br><br>
+Sum of operands: {1}<br><br><br><br><br>
+</font></p>
+<a href="/"><font face="Arial", font size="+1">
+Return to Instruction Page
+</font></a>
+"""
```

Comment:
Notice how similar the page is for each math function. Is there a way you could re-use code instead of having it in each function?